.env.example

```diff
@@ -0,0 +1,2 @@
+VITE_API=https://app-hrsei-api.herokuapp.com/api/fec2/hr-rfp/
+VITE_API_TOKEN=
```

Comment:
Make sure to copy this file to `.env` and fill this field in with your own GitHub token!